main/src/cgeo/geocaching/utils/DisplayUtils.java

```diff
@@ -48,15 +52,31 @@ public static int calculateNoOfColumns(final Context context, final float column
     }
 
     /**
-     * get actual height of given drawable resource
+     * get actual width and height of given drawable resource
      * @param res - resources to load from
      * @param resToFitIn - resource to check
-     * @return actual height
+     * @return actual width and height
      */
     public static Pair<Integer, Integer> getDrawableDimensions(final Resources res, @DrawableRes final int resToFitIn) {
         final Bitmap calc = BitmapFactory.decodeResource(res, resToFitIn);
         return new Pair<>(calc.getWidth(), calc.getHeight());
+    }
+
+    /**
+     * get actual width and height of given drawable
+     * @param resToFitIn - drawable to check
+     * @return actual width and height
+     */
+    public static Pair<Integer, Integer> getDrawableDimensions(final Drawable resToFitIn) {
+        return new Pair<>(resToFitIn.getIntrinsicWidth(), resToFitIn.getIntrinsicHeight());
+    }
+
+    public static Drawable getTintedDrawable(final Resources res, @DrawableRes final int menuRes, @ColorRes final int tintColor) {
```

Comment:
Problem was the action bar ...

That did not worked for me with changing fillColor...

But with the ongoing theme-rework that probably will change...

I guess it would be easy to change that...

Reply:
Not sure if it would help here, but at some point we need to move from `ActionBar` to `Toolbar` anyway...

Reply:
Then we will adapt / change that.